src/Screens/HomeScreen/HomeScreen.tsx

```diff
@@ -28,8 +31,8 @@ const HomeScreen = () => {
   );
 
   useEffect(() => {
-    dispatch(fetchTodos());
-  }, [dispatch]);
+    dispatch(fetchTodos(alert.show("Fetch todos")));
+  }, [dispatch, alert]);
```

Comment:
dispatch is enough, so remove alert from dependencies, because it is also constant.